### DISCUSSION TITLE
Folia support

### DISCUSSION THREAD

**Discussion by h14turkiye** [OmerBenGera/WildStacker#848] [Pending Feature Requests]
Adding support for PaperMC's new tech folia.

**Comment by OmerBenGera** [comment]
Folia is still under development and its API constantly changes, so I am not gonna do that until it is more stable :)

**Reply by h14turkiye** [reply]
If you are worried about API check out FoliaLib repo, any API changes we can just make a basic commit on FoliaLib and you wouldn't even touch your code.. You can use WrappedTask to adopt from BukkitTask. PaperMC wouldn't simply abondon Folia, and a stacker/spawner plugin is necessary for nearly every minecraft server type. And it should be thread safe since you already use ConcurrentHashMaps, Atomic types etc(I didn't look at your code fully, just skimmed it. I might be wrong).

Please reconsider it :pleading_face: , I tried to port in my computer but couldn't because of hardware limitations :cry:

**Comment by Jxagg** [comment]
any updates on this?
